Zainab Saud/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
+</head>
+<body>
+    <hearder>
+        <div class="wrapper">
+        <div class="logo">
+            <img src="./logo.png" alt="twitter logo">
+        </div>
+        <nav class="nav">
```

Comment:
you can trigger the tag  nav here directly without giving it a class with same name